### DISCUSSION TITLE
createModal not show on real device

### DISCUSSION THREAD

**Discussion by rnaufal32** [zepp-health/zeppos-forum#280] [Developer Feedback]
Hi, I am trying to create a modal for a confirmation dialog. The modal appears in the Amazfit Bip 5 simulator, but it does not appear on the device. I have attached the video. Please help.

Real device :
https://github.com/zepp-health/zeppos-forum/assets/33546066/f42452ce-25ef-4d39-801a-df2563216191

Simulator : 
https://github.com/zepp-health/zeppos-forum/assets/33546066/b67f8c04-c1df-461c-b90f-eb898ed6f009

**Comment by MyoungXUE** [comment]
Hello, this may be a bug in Amazfit Bip 5. I will confirm with relevant colleagues. If it is a bug, we will fix it in the OTA version.

**Reply by rnaufal32** [reply]
Okay. Thank you.

**Reply by MyoungXUE** [reply]
May I ask what is your firmware version number? I tested it on the latest firmware 3.7.0.1 and it works.

**Reply by MyoungXUE** [reply]
I found an error in the sample code of showModal, I will correct it as soon as possible, please refer to this usage

```js
import { createModal, MODAL_CONFIRM } from '@zos/interaction'

const dialog = createModal({
  content: 'hello world',
  autoHide: false,
  onClick: (keyInfo) => {
    const { type } = keyInfo
    if (type === MODAL_CONFIRM) {
      console.log('confirm')
    } else {
      dialog.show(false)
    }
  },
})

dialog.show(true)
```
